Frontend/src/Components/Notifications/Notifications.jsx

```diff
@@ -65,6 +66,42 @@ const Notifications = () => {
     }
   };
 
+  const renderNotifications = (notifications) => {
+    return (
+      <ul>
+        {notifications.map((notif, index) => {
+          const isListingNotif =
+            notif.type === "LIKE" ||
+            notif.type === "PURCHASE" ||
+            notif.type === "LIKE_PURCHASE";
```

Comment:
oo honestly in this case i would have isListingNotif as a function so like 
```
const getIsListingNotif(notif) {
return  notif.type === "LIKE" ||
            notif.type === "PURCHASE" ||
            notif.type === "LIKE_PURCHASE";
}
```

that way you can call getIsListingNotif here and on line 60